src/main/java/com/commercetools/sync/categories/helpers/CategorySyncStatistics.java

```diff
@@ -4,19 +4,25 @@
 import com.commercetools.sync.commons.helpers.BaseSyncStatistics;
 
 import javax.annotation.Nonnull;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
 
 import static java.lang.String.format;
 
 public class CategorySyncStatistics extends BaseSyncStatistics {
+
     /**
-     * Map that represents categories with missing parents; the keys of the map are the keys of the missing parent
-     * categories and the value of each is a list of the children category keys.
+     * Map that represents categories with missing parents.
+     *
+     * <ul>
+     *     <li>key: key of the missing parent category</li>
+     *     <li>value: a list of the parent's children category keys</li>
+     * </ul>
+     *
      */
-    private Map<String, List<String>> categoryKeysWithMissingParents = new HashMap<>();
+    private Map<String, List<String>> categoryKeysWithMissingParents = new ConcurrentHashMap<>();
```

Comment:
can't understand why here the implementation should be concurrent? The map is stored in `CategorySyncStatistics`, but updated over getter in some other place? Looks like wrong design.

Maybe just `getCategoryKeysWithMissingParents()` should return an immutable copy/view of `categoryKeysWithMissingParents` instance?

Reply:
@andrii-kovalenko-ct 
Now it is exactly changed like that: https://github.com/commercetools/commercetools-sync-java/pull/244/files#diff-c0fbda50bb0857abc8f4ea35c6239db4

It is also now a `ConcurrentMap<String, Set>` instead of `ConcurrentMap<String, List<String>>`

Also unit tests are added for the statistics class: https://github.com/commercetools/commercetools-sync-java/pull/244/files#diff-a89045e8bbd6c0998ee9220b42ca18ba